src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const tableRows = document.querySelector('tbody').rows;
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
```

Comment:
change the order, put the tableRows last, this way you could utilize tbody variable
`const tableRows = tbody.rows;`